src/renderer/shared/api/xcm/xcmService.ts

```diff
@@ -143,94 +181,115 @@ const getRelativeAssetLocation = (
   api: ApiPromise,
   amount: BN,
   assetLocation?: LocalMultiLocation,
-): VersionedMultiAsset | undefined => {
+): Object | undefined => {
   if (!assetLocation) return;
 
   const { parachainId: _, ...location } = assetLocation;
 
-  return api.createType('VersionedMultiAsset', {
-    V2: {
-      id: {
-        Concrete: {
-          parents: 0,
-          interior: Object.values(location).length ? createJunctionFromObject(location) : 'Here',
+  return {
+    V2: [
```

Comment:
It is not always V2, might be V1 as well, depending on a chain (V2 might not be present in VersionedMultiAsset on older versions of the pallet)